lib/zod/schemas/cluster.ts

```diff
@@ -32,6 +32,11 @@ const baseClusterSchema = z.object({
     description: "Description of the cluster",
     example: "Primary RISC-V prover",
   }),
+  zkvm_version_id: z.number().int().positive().openapi({
```

Comment:
main change on the API. Users now need to specify what zkvm/version is the cluster using